app/views/locations/index.html.erb

```diff
@@ -39,7 +39,7 @@
             <img
```

Comment:
I know this isn't really to do with this PR but can we add this alt text to the `<img>`?

`map showing the location of <<Location Name>> (link opens external website in new window)`